common/test/acceptance/tests/studio/test_studio_container.py

```diff
@@ -293,6 +294,25 @@ def test_edit_container_on_container_page(self):
         container = self.go_to_nested_container_page()
         self.modify_display_name_and_verify(container)
 
+    def test_edit_raw_html(self):
+        """
+        Test the raw html editing functionality.
+        """
+        modified_content = "<p>modified content</p>"
+
+        #navigate to and open the component for editing
+        unit = self.go_to_unit_page()
+        container = unit.xblocks[1].go_to_container()
+        component = container.xblocks[1].children[0]
+        component.edit()
+
+        html_editor = HtmlComponentEditorView(self.browser, component.locator)
+        html_editor.set_content_and_save(modified_content, raw=True)
+
+        #quick-n-dirty regex to strip simple html tags
+        stripped_content = re.sub('<[^<]+?>', '', modified_content)
```

Comment:
In a test, I would prefer that you just write out the expected string. Having a regex means that I have to understand how that will execute to know how the test is expecting things to work.

    stripped_content = "modified content"

At that point, you might as well just do it it in the assert

    self.assertEqual(component.student_content, "modified content")